wait/host_port.go

```diff
@@ -105,7 +107,7 @@ func buildInternalCheckCommand(internalPort int) string {
 	command := `(
 					cat /proc/net/tcp{,6} | awk '{print $2}' | grep -i :%x ||
 					nc -vz -w 1 localhost %d ||
-					/bin/bash -c '</dev/tcp/localhost/%d'
+					/bin/sh -c '</dev/tcp/localhost/%d'
```

Comment:
this is unnecessary, I believe. External command fix should be enough.

Reply:
Hey, sorry for the late response. I'll revert this change and resend. TBH, not sure why the travis tests fail tough.

Reply:
Mmmm, thinking about it, the internal command should be `sh`, because it's very likely that `bash` is not present in the container (i.e. Alpine-based images)